config/clusters/cloudbank/cluster.yaml

```
@@ -14,209 +14,166 @@ hubs:
     display_name: "Cloudbank Staging"
     domain: staging.cloudbank.2i2c.cloud
     helm_chart: basehub
-    helm_chart_values_files:
-      # The order in which you list files here is the order the will be passed
-      # to the helm upgrade command in, and that has meaning. Please check
-      # that you intend for these files to be applied in this order.
+      - common.values.yaml
```

I think the key got deleted by mistake
```suggestion
    helm_chart_values_files:
      - common.values.yaml
```